.idea/.idea.osu.Desktop/.idea/discord.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="DiscordProjectSettings">
+    <option name="show" value="ASK" />
+    <option name="description" value="" />
+  </component>
+</project>
```

Comment:
should not be committed. please remove

Reply:
Yeah, I'd realized right after pushing that this slipped into the commit. Didn't want to push this change until I received a review to avoid making unnecessary CI builds.